mvc/src/main/java/com/interface21/webmvc/servlet/view/JsonView.java

```diff
@@ -8,7 +11,23 @@
 
 public class JsonView implements View {
 
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
     @Override
     public void render(final Map<String, ?> model, final HttpServletRequest request, HttpServletResponse response) throws Exception {
+        response.setContentType(MediaType.APPLICATION_JSON_UTF8_VALUE);
+        String json = convertToJson(model);
+        response.getWriter().write(json);
+    }
+
+    private String convertToJson(Map<String, ?> model) throws JsonProcessingException {
+        if (model == null || model.isEmpty()) {
+            return "{}";
+        }
+        if (model.size() == 1) {
+            Object value = model.values().iterator().next();
```

Comment:
저도 무엇이 더 좋은지는 모르겠는데 우선 공유합니다.

```suggestion
            model.values()
                    .forEach(data -> writeJsonData(response, data));
```